joomla/joomla-cms
# XML Codestyle for the file application.xml

### Summary of Changes

XML Codestyle for the file `administrator/components/com_config/model/form/application.xml`
### Testing Instructions
- Apply this patch
- go to the global config
- confirm nothing is broken there
### Documentation Changes Required

none
